docs/sources/configure-client/go-span-profiles.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Go span profiles with trace-to-profiles
+sidebar_label: Go span profiles
+description: Go span profiles with trace-to-profiles
+weight: 100
+---
+
+# Go span profiles with trace-to-profiles
+
+![span-profiles screenshot](https://grafana.com/static/img/docs/tempo/profiles/tempo-profiles-Span-link-profile-data-source.png)
+
+## Prerequisites
+
+1. Profiling: You will need to have your application instrumented with Pyroscopes Go SDK. If you haven't done this yet, please refer to the [Go (push mode)]({{< relref "../configure-client/language-sdks/go_push" >}}) guide.
+2. Tracing: You will need to have your application instrumented with OpenTelemetry traces. If you haven't done this yet, please refer to the [OpenTelemetry](https://opentelemetry.io/docs/go/getting-started/) guide.
```

Comment:
Instead of Prerequisites, we use Before you begin. It's an equivalent title. 

```suggestion
## Before you begin

You need to have profiling and tracing set up before you can use span profiles. 

* Profiling: Your application must be instrumented with Pyroscopes Go SDK. If you haven't done this yet, please refer to the [Go (push mode)]({{< relref "../configure-client/language-sdks/go_push" >}}) guide.
* Tracing: Your application must be instrumented with OpenTelemetry traces. If you haven't done this yet, please refer to the [OpenTelemetry](https://opentelemetry.io/docs/go/getting-started/) guide.
```